test/bulkstatus.service.test.js

```diff
@@ -96,6 +96,21 @@ describe('checkBulkStatus logic', () => {
         expect(JSON.parse(response.text).message).toEqual('Could not find bulk export request with id: INVALID_ID');
       });
   });
+  test('check 429 error returned for spamming requests', async () => {
```

Comment:
Could use a test to make sure that after getting a 429, then appropriately waiting, you are able to get a 202 again.